src/pages/ProductDetail/DetailContainer.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import CartContainer from './DetailContainer/Cart';
+import Custom from './DetailContainer/Custom';
+
+const DetailContainer = props => {
```

Comment:
```suggestion
const DetailContainer = ({setProductImage, ~~~}) => {
```
매개변수자리에서 바로 구조분해 가능합니다.

Reply:
넵 구조분해 할당 해줬습니다.

Reply:
제눈에만 안보이나요?

Reply:
반영안됬습니다 코멘트 꼼꼼히 확인해주세요